src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -860,21 +896,26 @@ public function massEditAction(Request $request, $action)
             return $this->redirectToRoute('admin_product_catalog');
         }
 
-        /** @var ProductInterfaceProvider $productProvider */
+        /* @var $productProvider ProductInterfaceProvider */
```

Comment:
I changed these annotations volontarily in my last PR
why do you want to use this reverse annotation?
It's not very readable, it is inconsistent with the class or controller fields annotations
And I'm not even sure PHPStorm (or other IDEs) understand it

Reply:
Problem while rebase :) My bad